File: DSA/Array/largest.cpp
#include <iostream>
using namespace std;
int largestElement(int arr[], int n) {
    int largest=arr[0];
    for(int i=0;i<n;i++){
        if(arr[i]>largest){
            largest=arr[i];
        }
    }
    return largest;
}

int main() {
    int n;
    int arr[1000];
    cout<<"enter the number of elements:";
    cin>>n;
    cout << "Enter " << n << " elements:\n";
    for(int i=0;i<n;i++){
         cin >> arr[i];
    }
     cout << "Largest Element = " << largestElement(arr, n) << endl;
     return 0;
}

File: DSA/Array/largest.py
def largestinarray(arr):
    largest=arr[0]
    for i in range(len(arr)):
        if arr[i]> largest:
            largest=arr[i]
    return largest
if __name__=="__main__":
    arr1=[2, 5, 1, 3, 0]
    largesthere=largestinarray(arr1)
    print("The largest element in the array is :",largesthere)